README.md

```diff
@@ -170,7 +170,7 @@ To start coding and understand the frameworks, concepts and structure of the pro
 ## Experimental
 
 ### Make
-If you are willing to try using `make`, ensure you have installed git, node and couchdb (steps 1, 2 and 7 above), you may skip the rest.  This requires couchdb in the path to work correctly.
+If you are willing to try using `make`, ensure you have installed git, node and Couchdb (steps 1, 2 and 7 above), you may skip the rest.  This requires Couchdb in the path to work correctly.
```

Comment:
In order to be consistent with the rest of the doc, CouchDB should be referred to as `CouchDB` not `Couchdb`.  Also, since you are changing the text here, can you change `git` to `Git` and `node` to `Node.js` for consistency as well?